pkgs/by-name/md/mdfried/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  testers,
+}:
+
+rustPlatform.buildRustPackage rec {
```

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```
Lambdas are now possible and are preferred over `rec` attrsets